integration-tests/sdk/auth.it.js

```diff
@@ -104,7 +127,9 @@ describe('Auth Flows', () => {
 
       // check returned properties
       expect(tokenInfo).toHaveProperty('access_token')
-      expect(tokenInfo).toHaveProperty('scope', `manage_project:${projectKey}`)
+      expect(
+        modifiedTokenInfo(tokenInfo, ignoredResponseKeys[1])
```

Comment:
so why are we running tests towards a function implemented _inside_ the test?
How does this fix integration test?